planner/core/integration_partition_test.go

```diff
@@ -79,10 +79,10 @@ func TestListColVariousTypes(t *testing.T) {
 	tk.MustExec(`create table tstring (a varchar(32)) partition by list columns(a) (partition p0 values in ('a', 'b'), partition p1 values in ('c', 'd'))`)
 
 	err := tk.ExecToErr(`create table tdouble (a double) partition by list columns(a) (partition p0 values in (0, 1), partition p1 values in (2, 3))`)
-	require.Regexp(t, ".*not allowed.*", err)
+	require.Contains(t, err.Error(), "not allowed")
```

Comment:
```suggestion
	require.Error(t, err)
	require.Contains(t, err.Error(), "not allowed")
```

(and the rest of this file)